src/ARToolkitNFT.ts

```diff
@@ -269,32 +269,35 @@ export default class ARToolkitNFT {
     const targetPrefix = '/markerNFT_' + this.markerNFTCount++
     const extensions = ['fset', 'iset', 'fset3']
     let out;
-    var prefixes: any = [];
-    var pending = urls.length * 3;
-    pending -= 1;
-    if (pending === 0) {
-        const vec = new this.instance.StringList();
-        const markerIds = [];
-        for (let i = 0; i < prefixes.length; i++) {
-            vec.push_back(prefixes[i]);
-        }
-    /*var ret = Module._addNFTMarkers(arId, vec);
-    for (let i = 0; i < ret.size(); i++) {
-      markerIds.push(ret.get(i));
-    }*/
+    let prefixes: any = [];
+    let vec;
     for (var i = 0; i < urls.length; i++) {
```

Comment:
i think the loop should be a bit different, it load twice the marker because the url lenght is 1 (in this case of course that we load ONLY one marker) so it load the 0 and the 1 url, but it should load only the 0 address of the array.

Reply:
i don't think that this is the root of the issue.